README.md

```diff
@@ -1,214 +1,174 @@
+supported platform drivers are only compatible with Appium 2.
 
-### Requirements
+__[Please read the Migration Guide](https://appium.io/docs/en/latest/guides/migrating-1-to-2/) if you
+are still using Appium 1.__
 
-- macOS, Linux or Windows operating system
-- Node.js 16+. Any relevant [LTS release](https://nodejs.dev/en/about/releases/) should work just fine.
-- NPM (Node Package Manager) 8+
+If you use Appium Desktop or Appium Server GUI, you will not be able to upgrade to Appium 2, as
+both of these tools have been deprecated. Please use [Appium Inspector](https://github.com/appium/appium-inspector)
+in combination with a standalone Appium 2 server.
 
-These are only server requirements. Each driver might have its own requirements. Consider checking the corresponding driver tutorial for more details.
+## Installation
 
-### Server
+Appium can be installed using `npm` (other package managers are not currently supported). Please
+check the [installation docs](http://appium.io/docs/en/latest/quickstart/install/) for the
+system requirements and further information.
 
-To install Appium 2 server using Node Package Manager (npm) run the following command:
+If upgrading from Appium 1, make sure Appium 1 is fully uninstalled (`npm uninstall -g appium`).
+Unexpected errors might appear if this has not been done.
 
 ```bash
 npm i -g appium
 ```
 
-To update Appium 2 server using Node Package Manager (npm) run the following command:
-
- ```bash
-npm i -g appium@latest
-```
-
-:bangbang: Make sure Appium 1 is fully uninstalled before upgrading it to Appium 2 (`npm uninstall -g appium`). Unexpected errors might appear if this has not been done.
+Note that this will only install the core Appium server, which cannot automate anything on its own.
+Please install [drivers](#drivers) for your target platforms in order to automate them.
 
-### Drivers
+## Drivers
 
-Appium supports app automation across a variety of platforms, like iOS,
-Android, and Windows. Each platform is supported by one or more "drivers",
-which know how to automate that particular platform. Since version 2.0
-all drivers have been isolated from the Appium server app and can
-be managed independently using the [appium driver](https://appiumpro.com/editions/122-installing-appium-20-and-the-driver-and-plugins-cli) command line interface.
+Appium supports app automation across a variety of platforms, like iOS, Android, macOS, Windows,
+and more. Each platform is supported by one or more "drivers", which know how to automate that
+particular platform. You can find a full list of officially-supported and third-party drivers in
+[Appium Ecosystem's Drivers page](http://appium.io/docs/en/latest/ecosystem/drivers/).
 
-In general, the drivers management in Appium 2 is as simple as:
+Driver management is done using [Appium's Extension command-line interface](http://appium.io/docs/en/latest/cli/extensions/):
 
 ```bash
-# To install a new driver from npm
```

Comment:
I would rather keep the other installation options.

Reply:
I removed the local and github options here because they would be used a lot less often, but also added the docs link which also lists them. I guess I could add a comment to refer to the docs for other installation options?

Reply:
yes, the other options are mostly useful for extension developers or people who would like to get recent fixes.
A dedicated comment there makes sense to me.